wikipedia_tool.py

```
@@ -0,0 +1,255 @@
+        _, pred = torch.max(output, 1)
+
+    return jsonify({
+        "prediction": TOPICS[pred.item()]
+    })
+
+
+# =========================
+# RUN
+# =========================
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=5000)
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)